documentation/book/snip-external-logging-KafkaConnect.adoc

```diff
@@ -0,0 +1,12 @@
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: {kind}
```

Comment:
There's no point in using the attribute `{kind}` here, since this is necessarily the `KafkaConnect` kind.
```suggestion
kind: KafkaConnect
```